GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -55,18 +53,6 @@ public RevisionDiffControl()
 
         public void ForceRefreshRevisions()
```

Comment:
This method has only one callsite, so we can update the callsite to call `RefreshArtificial` and remove the method completely.

Reply:
It is not pointless to have two public interfaces here, but there will likely be some other reasoning at next change so I removed it